examples/streams/streams.description

```diff
@@ -1,3 +1,3 @@
-//The stream type represents a stream of events, which allows publishing events to the stream and subscribing to receive
-// events published to the stream.
-//Streams can be declared constrained by a type or as unconstrained streams which accept events of any type.
+//The `stream` type represents a stream of events, which allows publishing events to the stream and subscribing to receive
+// events that are published to the stream.
+//Streams can be declared as constrained by a type, or as unconstrained streams that accept events of any type.
```

Comment:
wouldn't it be better without the "as" in "Streams can be declared as constrained by a type"?

Basically the stream declaration is what would have the constraint.

Reply:
Rephrased it in commit https://github.com/ballerina-platform/ballerina-examples/pull/691/commits/2fcee42d0970ea300c76c5e48291229601695bce. 

Had second thoughts about the first sentence too so I reworded that a bit as well. Please check if  it is okay and gives the correct meaning.